.storybook/main.js

```diff
@@ -1,12 +1,38 @@
+const path = require('path');
+
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)",
-  ],
+  "stories": ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+
   "addons": [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/preset-scss",
     "storybook-addon-themes",
+    "@storybook/addon-webpack5-compiler-babel",
+    "@chromatic-com/storybook"
   ],
+
+  framework: {
```

Comment:
Is there any documentation we can point to for adding all of this configuration? Why was it not needed previously?

Reply:
Framework change was needed since we upgraded from webpack 4 to webpack 5 so we needed to mention that we are using webpack 5